README.md

```diff
@@ -170,6 +170,21 @@ const result = validate(value, validations)
 } */
 ```
 
+#### Format
+
+`format` (regex) -The format validator will validate a value against a regular expression of your chosing.
+
+```javascript
+const pattern = /^[0-9]{8}$/ // or you can use new RegExp('^[0-9]{8}$')
+const value = ''
```

Comment:
Would be better if the example wasn't a empty value. Usually I try to put something that is almost valid but still invalid